test/addons/presenter.test.js

```diff
@@ -1352,7 +1352,7 @@ describe('::send', function() {
       </Parent>
     )
 
-    wrapper.find('ActionButton').simulate('click')
+    wrapper.find(ActionButton).simulate('click')
```

Comment:
Names get minified, so test against the ActionButton component, not the string name.